documentation/asciidoc/computers/raspberry-pi/power-supplies.adoc

```diff
@@ -73,64 +96,60 @@ Exactly how much current (mA) the Raspberry Pi requires depends on what you conn
+The power requirements of the Raspberry Pi increase as you make use of the various interfaces on the Raspberry Pi. Combined, the GPIO pins can draw 50mA safely; each pin can individually draw up to 16mA. The HDMI port uses 50mA. The Camera Module requires 250mA. USB keyboards and mice can take as little as 100mA or as much as 1000mA. Check the power rating of the devices you plan to connect to the Raspberry Pi and purchase a power supply accordingly. If you're not sure, use an externally-powered USB hub.
 
-=== Typical power requirements
+You can check the status of power output to the USB ports using `vcgencmd`.
 
-|===
-| Product | Recommended PSU current capacity | Maximum total USB peripheral current draw | Typical bare-board active current consumption 
-
-|Raspberry Pi 1 Model A | 700mA | 500mA | 200mA 
-| Raspberry Pi 1 Model B |1.2A | 500mA | 500mA 
-| Raspberry Pi 1 Model A+ | 700mA | 500mA | 180mA
-| Raspberry Pi 1 Model B+ | 1.8A | 1.2A | 330mA 
-| Raspberry Pi 2 Model B | 1.8A | 1.2A | 350mA 
-| Raspberry Pi 3 Model B | 2.5A | 1.2A | 400mA 
-| Raspberry Pi 3 Model A+ | 2.5A | Limited by PSU, board, and connector ratings only | 350mA 
-| Raspberry Pi 3 Model B+ | 2.5A | 1.2A | 500mA 
-| Raspberry Pi 4 Model B | 3.0A | 1.2A | 600mA 
-| Raspberry Pi 400       | 3.0A | 1.2A | 800mA 
-| Raspberry Pi Zero | 1.2A | Limited by PSU, board, and connector ratings only | 100mA 
-| Raspberry Pi Zero W | 1.2A | Limited by PSU, board, and connector ratings only| 150mA 
-| Raspberry Pi Zero 2 W | 2A | Limited by PSU, board, and connector ratings only | 350mA
-|===
-
-From the Raspberry Pi B+ onwards, 1.2A is supplied to downstream USB peripherals. This allows the vast majority of USB devices to be connected directly to these models, assuming the upstream power supply has sufficient available current.
+[source,bash]
+----
+vcgencmd get_config usb_max_current_enable
+----
 
-Very high-current devices, or devices which can draw a surge current such as certain modems and USB hard disks, will still require an external powered USB hub. The power requirements of the Raspberry Pi increase as you make use of the various interfaces on the Raspberry Pi. The GPIO pins can draw 50mA safely (note that that means 50mA distributed across all the pins: an individual GPIO pin can only safely draw 16mA), the HDMI port uses 50mA, the Camera Module requires 250mA, and keyboards and mice can take as little as 100mA or as much as 1000mA! Check the power rating of the devices you plan to connect to the Raspberry Pi and purchase a power supply accordingly. If you're not sure, we would advise you to buy a powered USB hub.
-
-This is the typical amount of power (in amps) drawn by different Raspberry Pi models during standard processes:
+The following table describes the amount of power (in amps) drawn by different Raspberry Pi models during various workloads:
 
 |===
 | | | Raspberry Pi 1B+ | Raspberry Pi 2B | Raspberry Pi 3B | Raspberry Pi Zero | Raspberry Pi 4B 
```

Comment:
Did we also characterise Raspberry Pi 5 B, or do we not intend to / intend to later?

Reply:
I don't believe that this work has been done yet? Or at least I haven't seen numbers. @nathan-contino Can you follow up with the relevant folks and find out whether we have this data?

Reply:
Who should I ping about this? Not sure who did this testing in the first place. I can do some of the testing myself for the 5, but it might be a good idea to re-run these tests with the latest Raspberry Pi OS for each device.